### PR TITLE
Splitting PG and webwork2 codebase

### PR DESCRIPTION
**Pull request by pstaabp**
See openwebwork/pg#597 for description of this.

### CODE REVIEW

File: conf/webwork.apache2.4-config.dist [hunk 1]
```diff
@@ -75,7 +85,7 @@ die "webwork.apach2-config: Can't establish initial Course environment" unless r
 #  set important configuration variables (from defaults.config)
 
 my $webwork_url = $ce->{webwork_url};                    # e.g.  /webwork2
-my $pg_dir = $ce->{pg_dir};                              # e.g.  /opt/webwork/pg         
+# my $pg_dir = $ce->{pg_dir};                              # e.g.  /opt/webwork/pg
```
Comment (drgrice1) [inline conf/webwork.apache2.4-config.dist:88]: This is the line that is causing the conflict with the develop branch.

Also, don't comment this line out, just delete it.  You have moved the definition of `$pg_dir` above, so this is certainly not something that we want left here.

**Review by drgrice1 — COMMENTED** [review]
This looks good.  See my review at https://github.com/openwebwork/pg/pull/597#pullrequestreview-807863710.

Edit:  Of course my approval is pending you fixing the conflicts.

File: conf/webwork.apache2.4-config.dist (hunk 1 above)
Comment (pstaabp) [reply conf/webwork.apache2.4-config.dist]: This is deleted now and no merge conflicts.

File: conf/webwork.apache2.4-config.dist [hunk 2]
```diff
@@ -65,6 +65,16 @@ $WeBWorK::Constants::WEBWORK_DIRECTORY = $webwork_dir;
...
+
+# link to PG code libraries
+my $pg_dir ="/opt/webwork/pg";
+eval "use lib '$pg_dir/lib'"; die $@ if $@;
+$WeBWorK::Constants::PG_DIRECTORY = $pg_dir;
+$ENV{PG_ROOT} = $pg_dir;
+
+# moving WWSafe into $pg_dir/lib requires adding to lib
+
+eval "use lib '$pg_dir/lib'"; die $@ if $@;
```
Comment (taniwallach) [inline conf/webwork.apache2.4-config.dist:77]: This is the same as line 71, so probably should be deleted, and the comment on line 75 above could be before line 69.

Comment (drgrice1) [reply conf/webwork.apache2.4-config.dist:77]: Good catch @taniwallach.  I missed this.

@pstaabp:  Perhaps you could fix this in #1533.

**Pull request merged**